lib/greenhouse_api/client.rb

```diff
@@ -28,55 +28,51 @@ def initialize(api_key)
       @api_key = api_key
     end
 
-    def list_candidates(params = {})
-      list_many('candidates', params)
+    def candidates
+      Resources::Candidates
```

Comment:
I'd imagine this results in `client.candidates.list_candidates` when using it, which is awesome. It's slightly different from what I'd imagine the implementation would look like, in that the client would return an instance of the `Resources::Candidates` and `list_candidates` is an instance method. But your current approach doesn't seem to present any issue so feel free to keep it, we can always swap it out behind the scene.

Reply:
I like the modularity!
I think I would have expected to be able to consume the client this way
```ruby
client.list_candidates(...)
client.get_current_offer_for_application(...)
client.list_many(...)
client.create_user(...)
```

do you think it's possible to keep the instance method pattern (without going through the resource) while still preserving the separate class structure?

for example, if I call `client.list_candidates`, do you think that could be
```ruby
def list_candidates
    Resources::Candidates.list_candidates
end
```
? 

just trying to see how we can make life a little easier for the consumer 🤔

Reply:
ooo yes good suggestion!